netman/adapters/switches/juniper/base.py

```diff
@@ -633,7 +633,7 @@ def _push_interface_update(self, interface_id, configuration):
         try:
             self._push(configuration)
         except RPCError as e:
-            if "port value outside range" in e.message \
+            if "Port value outside range" in e.message \
```

Comment:
From the switch :

    <rpc-reply xmlns="urn:ietf:params:xml:ns:netconf:base:1.0" xmlns:junos="http://xml.juniper.net/junos/14.1X53/junos" xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:XXXXX">
    <rpc-error>
    <error-type>protocol</error-type>
    <error-tag>operation-failed</error-tag>
    <error-severity>error</error-severity>
    <error-message>
    port value outside range 0..127 for '128' in 'xe-0/0/128'
    </error-message>
    </rpc-error>
    <rpc-error>
    <error-type>protocol</error-type>
    <error-tag>operation-failed</error-tag>
    <error-severity>error</error-severity>
    <error-message>statement creation failed</error-message>
    <error-info>
    <bad-element>interface</bad-element>
    </error-info>
    </rpc-error>
    </rpc-reply>